.script/tests/KqlvalidationsTests/KqlValidationTests.cs

```diff
@@ -88,18 +130,18 @@ public void Validate_HuntingQueries_SkippedTemplatesDoNotHaveValidKql(string fil
         public void Validate_DetectionQueries_SkippedTemplatesDoNotHaveValidKql(string fileName, string encodedFilePath)
         {
             var res = ReadAndDeserializeYaml(encodedFilePath);
-            var id = (string) res["id"];
-        
+            var id = (string)res["id"];
+
             //Templates that are in the skipped templates should not pass the validation (if they pass, why skip?)
             if (ShouldSkipTemplateValidation(id) && res.ContainsKey("query"))
             {
-                var queryStr =  (string) res["query"];
+                var queryStr = (string)res["query"];
                 var validationRes = _queryValidator.ValidateSyntax(queryStr);
                 Assert.False(validationRes.IsValid, $"Template Id:{id} is valid but it is in the skipped validation templates. Please remove it from the templates that are skipped since it is valid.");

```

Comment:
Can we consider to put message in const and use it? I can see it has been used at multiple places.

Reply:
Updated the code with const and used it in all the methods.